app/views/admin/cycles/plugin_relations/petitions/index.html.slim

```diff
@@ -55,7 +55,7 @@
                | Versões anteriores
          div#past_versions style="display: none"
            ul.list-unstyled
-             - @petition.past_versions.each do |version|
+             - @petition.past_versions.order(created_at: :desc).each do |version|
```

Comment:
If we create an api, we have to remember this logic. You could use a order on the relation, or use a repository.